CalendarResponse.cs

```diff
@@ -1,105 +1,36 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-
-namespace TwilioSMSScheduler
-{
-    // Root myDeserializedClass = JsonConvert.DeserializeObject<Root>(myJsonResponse);
-    public class Creator
```

Comment:
I removed the classes and properties that the code wasn't using. This makes the code faster and easier to read.